android/build.gradle

```diff
@@ -215,6 +215,7 @@ repositories {
     maven {
         url "http://github.com/wada811/Android-Material-Design-Colors/raw/master/repository/"
     }
+    mavenCentral()
```

Comment:
Were you having issues resolving the new dependency without this? The top level build.gradle already defines `mavenCentral()` as a repo: https://github.com/the-blue-alliance/the-blue-alliance-android/blob/master/build.gradle#L41

Reply:
I did encounter dependency issues so I tried adding a few things and it worked when I added it worked. I'll try again to see if it was just Android Studio acting up.

Reply:
Thanks for catching this. It worked, I guess Android Studio was acting up when I ran it.